driver.py

```diff
@@ -0,0 +1,245 @@
+import getopt
```

Comment:
Include the "shebang": `#!/usr/bin/env python3` and mark this file as executable.